### ISSUE TITLE
Meteorism

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7773]
**Preferred term label:**

**Synonyms**
Tympanites

**Definition (free text, please give PubMed ID)**
An increase in the gas content inside the bowel associated with distension of the abdomen and hydroaeric noises. 

The most common symptom of meteorism is a bloated sensation. It may be associated with abdominal discomfort. Some individuals may also complain of a distended stomach and excessive belching and/or passage of flatus. 
PMID: 28613601

**Parent term (use hpo.jax.org/app)**
Abdominal symptom HP:0011458

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614616 

27/28  PMID: 22436048

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000319
- label: Meteorism
- parents: [Abdominal symptom (HP:0011458)](https://hpo.jax.org/app/browse/term/HP:0011458)
- pmid: PMID:28613601
- def: An increase in the gas content inside the bowel associated with distension of the abdomen and hydroaeric noises. The most common symptom of meteorism is a bloated sensation. It may be associated with abdominal discomfort. Some individuals may also complain of a distended stomach and excessive belching and/or passage of flatus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7773
- synonyms:
